### PR TITLE
New: Salesforce donation announcement

### PR DESCRIPTION
**Pull request by nzakas**
Announcement blog post about the Salesforce donation.

### CODE REVIEW

File: _posts/20202-08-06-salesforce-donates-10000-to-eslint.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Here's what Médédé Raymond KPATCHAA, who nominated ESLint, had to say:
+
+> "ESLint is an extremely useful tool for any JavaScript project. This tool allows us to set up consistent code formatting rules adopted by the whole team from the beginning of the project, greatly facilitates code reviews, and makes it easy to integrate new developers into the team. I know I am not alone in appreciating its flexibility and ease-of-use." -- Médédé Raymond KPATCHAA, Technical Architect, Salesforce
+
+We are grateful to Médédé for nominating ESLint this quarter and to the Salesforce employees who voted for ESLint. Financial support from companies like Salesforce is why ESLint has been able to keep up with the changing JavaScript ecosystem for the past seven years.
+
+[salesforce-post]: #todo
```
Comment (nzakas) [inline _posts/20202-08-06-salesforce-donates-10000-to-eslint.md:25]: I will replace this with the correct URL once they send me it.

File: _posts/20202-08-06-salesforce-donates-10000-to-eslint.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - Open Source
+  - Funding
+  - Open Collective
+  - Sponsors
+  - Backers
+---
+# Salesforce donates $10,000 to ESLint
+
+<p class="text-center">
+    <a href="https://indeed.com/" title="Salesforce" rel="noopener nofollow" target="_blank"><img class="lazyload" width="200" data-src="/assets/img/logos/salesforce.png" alt="Indeed" src="/assets/img/logos/salesforce.png"></a>
```
Comment (btmills) [inline _posts/20202-08-06-salesforce-donates-10000-to-eslint.md:13]:
```suggestion
    <a href="https://www.salesforce.com/" title="Salesforce" rel="noopener nofollow" target="_blank"><img class="lazyload" width="200" data-src="/assets/img/logos/salesforce.png" alt="Salesforce" src="/assets/img/logos/salesforce.png"></a>
```

Comment (nzakas) [reply _posts/20202-08-06-salesforce-donates-10000-to-eslint.md:13]: Oops, thanks!

File: _posts/20202-08-06-salesforce-donates-10000-to-eslint.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Here's what Médédé Raymond KPATCHAA, who nominated ESLint, had to say:
+
+> "ESLint is an extremely useful tool for any JavaScript project. This tool allows us to set up consistent code formatting rules adopted by the whole team from the beginning of the project, greatly facilitates code reviews, and makes it easy to integrate new developers into the team. I know I am not alone in appreciating its flexibility and ease-of-use." -- Médédé Raymond KPATCHAA, Technical Architect, Salesforce
+
+We are grateful to Médédé for nominating ESLint this quarter and to the Salesforce employees who voted for ESLint. Financial support from companies like Salesforce is why ESLint has been able to keep up with the changing JavaScript ecosystem for the past seven years.
+
+[salesforce-post]: #todo
```
Comment (nzakas) [inline _posts/20202-08-06-salesforce-donates-10000-to-eslint.md:25]:
```suggestion
[salesforce-post]: https://engineering.salesforce.com/giving-back-through-our-foss-fund-38754167c00d
```

**Pull request merged**